### PR TITLE
feat(ci): move monitoring job to orb (WIP)

### PR DESCRIPTION
**Pull request by rofe**
WIP for #38

### CODE REVIEW

**Comment by rofe** [comment]
@kptdobe any advice on how this orb could be tested without wreaking havoc on our monitoring setup? ;)

**Comment by kptdobe** [comment]
Set it up in a test service...

File: .circleci/orbs/helix-post-deploy/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+This orb contains centralized commands for use after deploying a new version of a Helix service.
+
+## Orb
+
+To integrate this orb into a CircleCI config, include the following line in `.circleci/config.yml` of a Helix repo:
+
+```yml
+orbs:
+  helix-post-deploy: adobe/helix-post-deploy@1.0.0
```
Comment (kptdobe) [inline .circleci/orbs/helix-post-deploy/README.md]:
```suggestion
  helix-post-deploy: adobe/helix-post-deploy@1.0.2
```

File: .circleci/orbs/helix-post-deploy/orb.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+              if [ "<< parameters.statuspage_group >>" != "" ]; then
+                spCmd="${spCmd} --group \"<< parameters.statuspage_group >>\""
+              fi
+              email="`eval ${spCmd}`"
+
+              # new relic automation
+              if [ "<< parameters.action_name >>" != "" ]; then
+                actionName="<< parameters.action_name >>"
+              else
+                actionName=`node -e "console.log(require('./package.json').name.replace('@adobe/helix-', ''))"`
```
Comment (kptdobe) [inline .circleci/orbs/helix-post-deploy/orb.yml]: wow :)
I am not a bash expert neither thus I am fine with those lines but I am sure there is a way to make that with a grep!

Comment (rofe) [reply .circleci/orbs/helix-post-deploy/orb.yml]: well, this line was copy/pasted from the original command - credits go to @trieloff ;)

**Pull request merged**